---
description: LOCALDB_ERROR_UNKNOWN_INSTANCE
title: LOCALDB_ERROR_UNKNOWN_INSTANCE | Документация Майкрософт
ms.custom: ''
ms.date: 03/03/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: performance
ms.topic: reference
ms.assetid: 67bca4f1-915d-46d4-9293-3742c16930a1
author: WilliamDAssafMSFT
ms.author: wiassaf
ms.openlocfilehash: ee0ffae1ba3420b5f4a182555d365cd3b77f620e
ms.sourcegitcommit: 0e0cd9347c029e0c7c9f3fe6d39985a6d3af967d
ms.translationtype: MT
ms.contentlocale: ru-RU
ms.lasthandoff: 12/02/2020
ms.locfileid: "96506107"
---
# <a name="localdb_error_unknown_instance"></a>LOCALDB_ERROR_UNKNOWN_INSTANCE
 [!INCLUDE [SQL Server](../../includes/applies-to-version/sqlserver.md)]
    
## <a name="details"></a>Сведения  
  
| attribute | Значение |
| --------- | ----- |
|Название продукта|SQL Server|  
|Идентификатор события|263|  
|Источник события|Среда выполнения локальной базы данных SQL Server 12.0|  
|Компонент|API среды выполнения локальной базы данных|  
|Текст сообщения|Указанный экземпляр локальной базы данных не существует.|  
  
## <a name="explanation"></a>Объяснение  
 Указанный экземпляр не существует.  
  
## <a name="user-action"></a>Действие пользователя  
 Проверьте правильность указанного имени экземпляра.  
  
  
